### ISSUE TITLE
Removal Request

### ISSUE THREAD

**Issue by columndeeply** [columndeeply/hosts#17]
Please post here any domains you think should be removed from the blocklist

**Comment by anders-m** [comment]
Dr.dk (Danish television website)
Sensibo.com (smart indoor climate control)

**Comment by anders-m** [comment]
mitid.dk (Danish digital ID service)

**Comment by columndeeply** [comment]
Hi @anders-m , could you confirm which hosts file has those domains? I can't seem to find them on any of the lists.

**Comment by TPS** [comment]
https://indiatimes.com/ is a well-known news site.

**Comment by columndeeply** [comment]
> https://indiatimes.com/ is a well-known news site.

Thanks for the report @TPS, it's been whitelisted in the latest release.

**Comment by ldidry** [comment]
https://stats.framasoft.org is a Matomo (analytics) instance, not a porn weasite.

**Comment by anders-m** [comment]
> [@columndeeply](https://github.com/columndeeply)
> 
> Dr.dk (Danish television website) Sensibo.com (smart indoor climate control)

Strange, now both of them is'nt on the list. 

Thanks for your work and reply!

**Comment by gaytomycode** [comment]
*.gaytomycode.com. you have many of my subdomains blocked

**Comment by columndeeply** [comment]
Added to the whitelist and notified the upstream lists, thanks for letting me know! @gaytomycode

**Comment by ldidry** [comment]
Ping @columndeeply, could you whitelist stats.framasoft.org, please? It’s not, in any way, related to a porn website. It’s a Matomo instance for a french non-profit association which purpose is popular education on digital challenges, [Framasoft](https://framasoft.org).

**Comment by columndeeply** [comment]
@ldidry done! Thanks for letting me know

**Comment by ldidry** [comment]
Thank you :-)

**Comment by columndeeply** [comment]
@kewlguy781 thanks for letting me know. I have whitelisted it and notified the upstream repos that had it listed.

**Comment by RoboticEarthling** [comment]
www.nakedbiblepodcast.com
There is no NSFW content there. (But I definitely understand the mix-up!)

**Comment by SeanSith** [comment]
graph.facebook.com - Breaks, among other things, Kids Messenger. Not sure why it would be on a porn blocking list in the first place, though.

**Comment by columndeeply** [comment]
@SeanSith thanks, whitelisted.

**Comment by ggalmeida0** [comment]
Should `googleusercontent.com` be in there?
The images generated by [gemini](https://gemini.google.com/) seem to be hosted from that domain, with that blocked they never show up.

**Comment by ggalmeida0** [comment]
> datadog-anything seems to be related to their analytics/tracking. (I don't trust even their NTP servers.) I've been running datadog$important for years w/ no negative effects.


Agree, datadog is a good general block. Though I'm noticing that domain in particular is serving images for some websites ([eg](https://www.butcherbox.com/)). Thought it might not be appropriate for a porn list.

**Comment by TPS** [comment]
> Agree, datadog is a good general block. Though I'm noticing that domain in particular is serving images for some websites ([eg](https://www.butcherbox.com/)).

@ggalmeida0 I tried whatever I could on your <kbd>[eg](https://www.butcherbox.com/)</kbd> site & I didn't noticed _any_ incorrect blocking, especially of `datadog`-variety.  Do you have a specific link where I can see such?  Else, I support blocking such wholesale, as above.

**Comment by TPS** [comment]
> the images are coming from the `ctfassets.net` domain not the datadog one.

@ggalmeida0 How are you getting _that_ domain blocked?  It's not from this list!

**Comment by Wendelvsouza** [comment]
Hello, I would like to have access to the reddit forum, apparently the lists block this forum. Could you remove this forum from the list?  Or put it in a separate link? Thank you @columndeeply

**Comment by qlofly** [comment]
yandex.by - is the same as blocking google.com
It's just a search engine.

**Comment by columndeeply** [comment]
@qlofly done. I wasn't aware yandex had country specific domains. I've whitelisted .by, .kz, .lv and .uz

**Comment by nathan-d** [comment]
Please remove kobo.com from **hosts04** - Blocks Rakuten ebook store used by Kobo ereaders.

**Comment by columndeeply** [comment]
@Just-a-hamster-that-codes @nathan-d done and done! I'll notify upstream so they can whitelist them too. Thanks for letting me know.

**Comment by Castillu84** [comment]
Please remove jellyfin.org from hosts01 - Blocks a Free Software Media System.

**Comment by columndeeply** [comment]
@Castillu84 done! Thanks for letting me know.

**Comment by TPS** [comment]
@Castillu84 Some of those non-domain would work fine for software _not_ bound solely to domain names (e.g., most mobile adblockers), but others (`adult`⁈) are likely overbroad to be useful.

**Comment by columndeeply** [comment]
@nathan-d done!

@Castillu84 Thanks! Yes, those shouldn't be there, I think I pasted some stuff wrong when adding new domains... I've added a filter to ensure all lines have a valid URL.

**Comment by Squidroot2** [comment]
oisd.nl is a site that hosts blocklists and it should be whitelisted. The following domains are currently blocked:
- includecheck.oisd.nl
- nsfw-check.oisd.nl
- nsfw_check.oisd.nl
- oisd.nl
- test.oisd.nl
- zzznsfw.test.oisd.nl
- zzztest.oisd.nl
